src/librustdoc/html/highlight/tests.rs

```diff
@@ -1,82 +1,27 @@
-use rustc_session::parse::ParseSess;
-use rustc_span::edition::Edition;
-use rustc_span::with_session_globals;
-use rustc_span::FileName;
+use expect_test::expect_file;
```

Comment:
I think it'd be better to move this into a `test` folder and not in the middle of the source code (since you're changing a lot of things... ;) )